thorlcr/activities/choosesets/thchoosesetsslave.cpp

```diff
@@ -367,7 +367,7 @@ class ChooseSetsLastActivity : public ChooseSetsPlusActivity
         for (unsigned idx=0; idx < numSets; idx++)
         {
             rowcount_t firstToCopy = 0;
-            ::ActPrintLog(this, thorDetailedLogLevel, "CHOOSESETSLAST: %d P(%" RCPF "d) C(%" RCPF "d) L(%" I64F "d) T(%" RCPF "d)", idx, priorCounts[idx], counts[idx], limits[idx], totalCounts[idx]);
+            ::ActPrintTrace(this, "CHOOSESETSLAST: %d P(%" RCPF "d) C(%" RCPF "d) L(%" I64F "d) T(%" RCPF "d)", idx, priorCounts[idx], counts[idx], limits[idx], totalCounts[idx]);
```

Comment:
question: the intention is to change all other ActPrintLog* functions to ActPrintTrace I assume?

Reply:
No - trace implies "Only if asked for" while log is "always" (we may want better names). The variety of ActPrintLog that took a detail level always used the higher level...